examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,57 @@
+"""
+=============================
+Plot positions on a blank map
+=============================
+This example showcases how to plot positions on a blank map.
+It is commonly seen in papers and presentations to show HPC positions of events that occurred at different times and therefore no single observation is appropriate.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+################################################################################
+# First we will create a blank map using with an array of zeros.
+# Since there is no WCS information, we will need to construct a header to pass to Map.
+data = np.zeros((1000, 1000))
+
+# Define coordinates and frame of reference and make the header using sunpy.map.make_fitswcs_header
+skycoord = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime='2013-10-28',
+                    observer='earth', frame=frames.Helioprojective)
+header = sunpy.map.make_fitswcs_header(data, skycoord)
```

Comment:
```suggestion
header = sunpy.map.make_fitswcs_header(data, skycoord, scale=[2,2]*u.arcsec/u.pixel)
```

If you want the solar limb to be in the field of view (i.e. so you can use blank_map.draw_limb() and see it) then you need either the data array to be larger (the radius of Sun is ~960'') or you can set the pixel scale to be two arcsec per pixel as suggested here.